app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def recent_comments(id)
+    Comment.where(post_id: id).last(5)
+  end
```

Comment:
- [ ] It will be great to just return the comments. Remember there is already a setup association between the comments and the post. So all you need to do is `comments.order('created_at Desc').limit(5)` first you order the comments then get the 5 most recent